### PR TITLE
Added some learning resource to README.md

### PR DESCRIPTION
**Pull request by CLOEI**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ Well, the idea to use useEffect hook is to execute code that needs happens durin
 
 ### Learning resources:
 <!-- Add learning resources here -->
+[Using the Effect Hook](https://reactjs.org/docs/hooks-effect.html)
```
Comment (saurabhnative) [inline README.md:22]: We can remove this resource since it has same link as the resource mentioned on line no.24

**Comment by saurabhnative** [comment]
@CLOEI Added comment to pull request. Please resolve the issue and comment back on the conversion with changes made to the Pull Request.

**Comment by CLOEI** [comment]
@saurabhnative Hello, my bad i didnt notice that i havent edited the link, its fixed now

**Comment by saurabhnative** [comment]
@all-contributors please add @CLOEI for docs

**Comment by saurabhnative** [comment]
@CLOEI I have merged the pull request with `hacktoberfest-accepted` label as per official guidelines for maintainers for Hacktoberfest.
The PR should get reflected in your Hacktoberfest profile in about 1-2 hours and will get accepted as valid PR after 14 days review window from hacktoberfest team.
Thanks for your contributions.

**Pull request merged**